vllm/v1/sample/sampler.py

```diff
@@ -151,7 +151,7 @@ def gather_logprobs(
                                                  dim=-1)
 
         # Get with the logprob of the prompt or sampled token.
-        token_ids = token_ids.unsqueeze(-1)
+        token_ids = token_ids.unsqueeze(-1).to(torch.long)
```

Comment:
nit: .long()

Reply:
Just curious: How is it different from `.to(torch.long)`?

Reply:
it's the same. my reason is long() is shorter, :-)

Reply:
Reference: https://pytorch.org/docs/stable/generated/torch.Tensor.long.html#torch.Tensor.long